modules/kubernetes-addons/variables.tf

```diff
@@ -1,5 +1,6 @@
 variable "eks_cluster_id" {
   description = "EKS Cluster Id"
+  default     = ""
```

Comment:
Don't set the empty value. This is a mandatory value for this add-on

Reply:
> I am wondering if the sorting can be done with pre-commit? Nice to have this in the workflows

the problems will be the more complex maps

Reply:
> Don't set the empty value. This is a mandatory value for this add-on

your PR should have address this... i'll drop all of my changes